tests/system/action/base.py

```diff
@@ -53,6 +57,25 @@ def request_json(self, payload: Payload, anonymous: bool = False) -> Response:
         client = self.client if not anonymous else self.anon_client
         return client.post("/", json=payload)
 
+    def execute_action(
```

Comment:
I thought `execute_action` (which only executes the action and does no stuff around it) vs. `request` (which does a full request with all kinds of required stuff around it) was a sufficent difference. But if you want, I can rename it. I would suggest `execute_action_internally` (so basically your first suggestion) since the latter suggests that you can only execute internal actions with it.

Reply:
Both solutions are okay for me:D